### PR TITLE
onboarding: update outline a bit based on what we've learned, add language about measurement times

### PR DESCRIPTION
**Pull request by nicks**
Hello @ellenkorbes, @maiamcc,

Please review the following commits I made in branch nicks/ch5311/guides:

28d43d69cb9668d8c6ad196fbaf81fde204ada53 (2020-03-12 10:43:41 -0400)
onboarding: update outline a bit based on what we've learned, add language about measurement times

Code review reminders, by giving a LGTM you attest that:

* Commits are adequately tested
* Code is easy to understand and conforms to style guides
* Incomplete code is marked with TODOs
* Code is suitably instrumented with logging and metrics

### CODE REVIEW

File: onboarding_guide_outline.md [hunk 1]
```diff
@@ -67,15 +67,31 @@ Link to snapshot of the server running
...
 Let's see it!
 
+Screenshot
+
 Current score of our naive hello world app.
 
-### Attempt #1
+You may notice that the time displayed in the Tilt sidebar is different than the
+time our app logged. That's OK! In multi-service development, there are many
+"start times" we care about -- time to build, time to schedule, time until
```
Comment (maiamcc) [inline onboarding_guide_outline.md]: nice, I like this! my one quibble is that it's less "start time" and more "duration" or "benchmark"?

File: onboarding_guide_outline.md [hunk 2]
```diff
@@ -67,15 +67,31 @@ Link to snapshot of the server running
 
 Before we try to make this faster, let's measure it.
 
-Add a local_resource that records the current time, and kicks off a build.
+Tilt can run commands locally.
```
Comment (maiamcc) [inline onboarding_guide_outline.md]: This whole section of the template seems scattered. e.g. this line isn't helpful without context. i'd strike it or replace with: "in addition to [running things]|[doing magic] in your cluster, Tilt also gives you tools to run commands locally."

Consider:
> In addition to [running things]|[doing magic] in your cluster, Tilt also gives you tools to run commands locally.

> We want to measure the time from code change to new process. To do that, we'll be making use of [local_resource](link-to-doc), which lets you locally run scripts, shell code, or servers, and manage them from your sidebar like any other Tilt resource.

> First, we add...

Comment (nicks) [reply onboarding_guide_outline.md]: oh, ya, totes agree, this is intended to be an outline that hits the key story beats; I expect an outline to be pretty scattered! Added some parens on this

File: onboarding_guide_outline.md (hunk 1 above)
Comment (nicks) [reply onboarding_guide_outline.md]: ya, i like benchmark better too

File: onboarding_guide_outline.md (hunk 2 above)
Comment (maiamcc) [reply onboarding_guide_outline.md]: sure, I just think that this bit is consistent enough across guides that we can provide the actual copy

**Pull request merged**